ui/components/MesheryPatterns.js

```diff
@@ -1478,7 +1478,7 @@ function MesheryPatterns({
             pattern={selectedPattern.pattern}
           />
         )}
-        <div className={StyleClass.toolWrapper}>
+        <div className={StyleClass.toolWrapper} style={{ zIndex: 125 }}>
```

Comment:
@captain-Akshay you need to pass this style to root component of toolWrapper where its defined because this change will have effect on only pattern section not others